HerPublicWebsite/Controllers/StaticPagesController.cs

```diff
@@ -1,15 +1,29 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Hosting;
 
 namespace HerPublicWebsite.Controllers;
 
 public class StaticPagesController : Controller
 {
+    private readonly IWebHostEnvironment environment;
+
+    public StaticPagesController(IWebHostEnvironment environment)
+    {
+        this.environment = environment;
+    }
+
     [HttpGet("/")]
     public IActionResult Index()
     {
+        if (environment.IsProduction())
+        {
+            return Redirect(Constants.SERVICE_URL);
+        }
```

Comment:
Rationale here is that in non-prod we want to host our own landing page, for testing, but in production we redirect to the gov.uk one.